Source/RunActivity/Viewer3D/WebServices/Web/HUD/index.js

```diff
@@ -126,6 +128,10 @@ function ApiHeadUpDisplay() {
 								locomotivetype = true;
 								Str += "<td colspan='2' >" + obj.extraTable.values[next] + "</td>";
 							}
+							else if (colspan2)
+							{
+								Str += "<td class='td_nowrap' colspan='2' >" + obj.extraTable.values[next] + "</td>";
+							}
```

Comment:
For the rows that have per-axle values (in the Forces table).

Reply:
Agreed